onnxruntime/server/util.cc

```diff
@@ -44,5 +52,13 @@ protobufutil::Status GenerateProtobufStatus(const onnxruntime::common::Status& o
   return protobufutil::Status(code, oss.str());
 }
 
+namespace internal {
```

Comment:
When I first read through this code I thought `internal` was some namespace from grpc. I suggest a more informative namespace under onnxruntime::server::

Reply:
maybe I should fully qualify it in my implementation otherwise, whats your suggestion?

Reply:
Ah you don't need to fully qualify. serverutil?